spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/SpannerConverters.java

```diff
@@ -80,38 +81,63 @@ private SpannerConverters() {
 					};
 
 	/**
-	 * A converter from {@link java.util.Date} to the Spanner date type.
+	 * A converter from {@link LocalDate} to the Spanner date type.
 	 */
 	// @formatter:off
-	public static final Converter<java.util.Date, com.google.cloud.Date>
-					JAVA_TO_SPANNER_DATE_CONVERTER = new Converter<java.util.Date, com.google.cloud.Date>() {
+	public static final Converter<LocalDate, com.google.cloud.Date>
+					LOCAL_DATE_TIMESTAMP_CONVERTER = new Converter<LocalDate, com.google.cloud.Date>() {
 		// @formatter:on
 		@Nullable
 		@Override
-		public com.google.cloud.Date convert(java.util.Date date) {
-			Calendar cal = Calendar.getInstance();
-			cal.setTime(date);
-			return com.google.cloud.Date.fromYearMonthDay(cal.get(Calendar.YEAR), cal.get(Calendar.MONTH) + 1,
-							cal.get(Calendar.DAY_OF_MONTH));
+		public com.google.cloud.Date convert(LocalDate date) {
+			return com.google.cloud.Date.fromYearMonthDay(date.getYear(), date.getMonthValue(), date.getDayOfMonth());
 		}
 	};
 
 	/**
-	 * A converter from the Spanner date type to {@link java.util.Date}.
+	 * A converter from the Spanner date type to {@link LocalDate}.
 	 */
 	// @formatter:off
-	public static final Converter<com.google.cloud.Date, java.util.Date> SPANNER_TO_JAVA_DATE_CONVERTER =
-					new Converter<com.google.cloud.Date, java.util.Date>() {
+	public static final Converter<com.google.cloud.Date, LocalDate> TIMESTAMP_LOCAL_DATE_CONVERTER =
+					new Converter<com.google.cloud.Date, LocalDate>() {
 						// @formatter:on
 						@Nullable
 						@Override
-						public java.util.Date convert(com.google.cloud.Date date) {
-							Calendar cal = Calendar.getInstance();
-							cal.set(date.getYear(), date.getMonth() - 1, date.getDayOfMonth());
-							return cal.getTime();
+						public LocalDate convert(com.google.cloud.Date date) {
+							return LocalDate.of(date.getYear(), date.getMonth(), date.getDayOfMonth());
 						}
 					};
 
+	/**
+	 * A converter from {@link java.util.Date} to the Spanner timestamp type.
+	 */
+	// @formatter:off
```

Comment:
Out of curiosity, what does the formatter do if you leave this out?

Reply:
it tries to put everything on one line